js/inventory.js

```diff
@@ -0,0 +1,45 @@
+import * as utilityScript from './utility.js';
+
+let isInventoryOpen = false;
+
+document.querySelector(".inventorySlot.slotArrow").onclick = function() {
+    if (!isInventoryOpen) {
+        moveInventory("-", 85, 0, 0);
+        isInventoryOpen = true;
+        utilityScript.unhideElement(document.querySelector(".blackoutPanel"))
+    } else if (isInventoryOpen) {
+        moveInventory("+", 40, 180, 0.5);
+        isInventoryOpen = false;
+        setTimeout(() => {
```

Comment:
Minor: But fix it in the next your PRs
```suggestion
const panel = document.querySelector(".blackoutPanel");

if (!isInventoryOpen) {
  ....
   utilityScript.unhideElement(panel);
} else { // remove if check
   ....
   setTimeout(() => utilityScript.hideElement(panel), 400);
}